src/Search/Request/RequestConfiguration.php

```diff
@@ -82,6 +84,41 @@ public function getPage(): int
         return (int) $this->request->get('page', 1);
     }
 
+    /**
+     * @SuppressWarnings(PHPMD.CyclomaticComplexity)
+     */
+    public function manageRangeField(string $field): void
+    {
+        $price = $this->request->get($field, []);
+        if (!\is_array($price) || empty($price)) {
+            return;
+        }
+
+        /** @var array $price */
+
+        // Reverse min and max if min is greater than max
+        if (isset($price['min'], $price['max'])) {
+            $min = (int) $price['min'];
+            $max = (int) $price['max'];
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** `$price['min']` and `$price['max']` can't be floats ?‏

Reply:
You are right ! I casted it on float.
The value switch is on the original values, not the casted one to avoid issue if I convert the value to float, then to string.

The float value is only used for comparison